### PR TITLE
libteam: Set buffer size of netlink cli sockets

### PR DESCRIPTION
**Pull request by stcheng**
Signed-off-by: Shu0T1an ChenG <shuche@microsoft.com>

### CODE REVIEW

**Comment by stcheng** [comment]
retest this please

File: src/libteam/0006-libteam-Set-buffer-size-of-netlink-cli-sockets.patch [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+diff --git a/libteam/libteam.c b/libteam/libteam.c
+index 106e5cf..f7010b9 100644
+--- a/libteam/libteam.c
++++ b/libteam/libteam.c
+@@ -600,6 +600,17 @@ int team_init(struct team_handle *th, uint32_t ifindex)
+ 		return -nl2syserr(err);
+ 	}
+ 
++    err = nl_socket_set_buffer_size(th->nl_cli.sock, 98304, 0);
```
Comment (lguohan) [inline src/libteam/0006-libteam-Set-buffer-size-of-netlink-cli-sockets.patch]: why is this value? a magic number?

File: src/libteam/0006-libteam-Set-buffer-size-of-netlink-cli-sockets.patch [hunk 2]
```diff
@@ -0,0 +1,38 @@
+From 12528607db866fe84b870127de64d40efb0a98a0 Mon Sep 17 00:00:00 2001
+From: Shu0T1an ChenG <shuche@microsoft.com>
+Date: Mon, 14 Jan 2019 16:58:36 -0800
+Subject: [PATCH] Set buffer size of netlink cli sockets
+
+Set both netlink cli sock and netlink cli event sock receive buffer
+size to 98304.
```
Comment (lguohan) [inline src/libteam/0006-libteam-Set-buffer-size-of-netlink-cli-sockets.patch]: can you decribe the problem trying to address in this patch?

File: src/libteam/0006-libteam-Set-buffer-size-of-netlink-cli-sockets.patch (hunk 1 above)
Comment (yxieca) [reply src/libteam/0006-libteam-Set-buffer-size-of-netlink-cli-sockets.patch]: Set buffer size failed on my DUT. Does this value rely on certain sysctl setting?

**Pull request closed without merging**